web_config/webconfig.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/python
+
+try:  # Python2
```

Comment:
xonsh doesn't not support python-legacy (a.k.a. python2), you can assume python3 is used.